config/initializers/sidekiq.rb

```diff
@@ -10,16 +11,17 @@
 end
 
 if Rails.env.production?
-  ActiveSupport.on_load(:active_record) do
-    unless Sidekiq::Cron::Job.find('Reminders')
-      Sidekiq::Cron::Job.create(
-        name: 'Reminders', cron: '0 9-20 * * *', class: 'RemindWorker'
-      )
-    end
-    unless Sidekiq::Cron::Job.find('Shift Surveys')
-      Sidekiq::Cron::Job.create(
-        name: 'Shift Surveys', cron: '0 5,20,35,50 * ? * *', class: 'ShiftSurveyWorker'
-      )
-    end
-  end
-end
\ No newline at end of file
+  CRONJOBS = [
+  {
+    'name'  => 'Reminders',
+    'class' => 'RemindWorker',
+    'cron'  => '0 9-20 * * *',
+  },
+  {
+    'name'  => 'Shift Surveys',
+    'class' => 'ShiftSurveyWorker',
+    'cron'  => '0 5,20,35,50 * ? * *'
+  }]
+
+  Sidekiq::Cron::Job.load_from_array! CRONJOBS
```

Comment:
Looks really good @creaturenex! My suggestions here are style only.

```suggestion
  CRONJOBS = [
    {
      'name'  => 'Reminders',
      'class' => 'RemindWorker',
      'cron'  => '0 9-20 * * *'
    },
    {
      'name'  => 'Shift Surveys',
      'class' => 'ShiftSurveyWorker',
      'cron'  => '0 5,20,35,50 * ? * *'
    }
  ]

  Sidekiq::Cron::Job.load_from_array!(CRONJOBS)
```